### PR TITLE
[KC-4] Capture image

### PR DESCRIPTION
**Pull request by JohnStuartBarrie**
https://kyanmedia.atlassian.net/browse/KC-4

### CODE REVIEW

File: src/view/kc-display.js [hunk 1]
```diff
@@ -4,8 +4,9 @@ export class KCDisplay extends LitElement {
   render () {
     return html`
     <div>
-      <kc-webcam-selector></kc-webcam-selector>
+      <kc-webcam-selector ></kc-webcam-selector>
```
Comment (dquilter) [inline src/view/kc-display.js]: Unneeded space?

File: src/view/kc-transport.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import { connect } from 'pwa-helpers'
+import { LitElement, html } from 'lit-element'
+import { store } from '../model/store.js'
+import { capture } from '../action/capture.js'
+
+export class Transport extends connect(store)(LitElement) {
```
Comment (dquilter) [inline src/view/kc-transport.js]: There's a bit too much going on in this line for my liking...

File: src/view/kc-transport.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  }
+
+  onClick (ev) {
+    capture();
+  }
+
+  render () {
+    return html`
+      <style>
+        :host { display: block; }
```
Comment (dquilter) [inline src/view/kc-transport.js]: I probs shouldn't be doing this in a PR, but what's `:host`?

File: src/view/kc-webcam.js [hunk 4]
```diff
@@ -3,15 +3,9 @@ import { LitElement, html } from 'lit-element';
 import { store } from '../model/store.js'
 
 export class KCWebCam extends connect(store)(LitElement) {
```
Comment (dquilter) [inline src/view/kc-webcam.js]: There's a bit too much going on in this line for my liking...

File: src/view/kc-transport.js (hunk 2 above)
Comment (JohnStuartBarrie) [reply src/view/kc-transport.js]: Give a suggestion, please.

File: src/view/kc-display.js (hunk 1 above)
Comment (JohnStuartBarrie) [reply src/view/kc-display.js]: Noticed it and brutally commited without self reviewing and fell asleep. Will correct.

File: src/view/kc-transport.js (hunk 3 above)
Comment (JohnStuartBarrie) [reply src/view/kc-transport.js]: As I understand it is the CSS shadowDom. I'll remove and we can have a look at that later. As you can see style, I haven't been styling.

File: src/view/kc-webcam.js (hunk 4 above)
Comment (JohnStuartBarrie) [reply src/view/kc-webcam.js]: Suggestion, please.

Comment (JohnStuartBarrie) [reply src/view/kc-webcam.js]: Is it the `export` that you don't like? Prefer it at the end?

**Review by dquilter — APPROVED** [review]

**Pull request merged**